modules/custom/openy_analytics/src/AnalyticsCron.php

```diff
@@ -118,7 +118,7 @@ function getServerInfo() {
     $db_detailed_version = $this->database->query("SHOW VARIABLES LIKE '%version%';")
       ->fetchAllKeyed();
     $server_software = $_SERVER['SERVER_SOFTWARE'];
-    $php_version = $_SERVER['PHP_VERSION'];
+    $php_version = phpversion();
```

Comment:
Note that the version string returned by phpversion() may include more information than expected: "5.5.9-1ubuntu4.17", for example.
From https://www.php.net/phpversion
Are we fine with having this string as is?

Reply:
I can answer.
Yes, we are good
Thank you )))